src/module/entities/helper.js

```diff
@@ -1433,6 +1419,11 @@ module.exports = class UserProjectsHelper {
 							entityCreation.message = CONSTANTS.apiResponses.FIELD_MISSING
 							return entityCreation
 						}
+
+						if(entityCreation.metaInformation.externalId){
```

Comment:
@MallanagoudaB hence both data is metaInformation.externalId see if you can optimise this code block